Block/Adminhtml/System/Config/Form/Field/Information.php

```diff
@@ -33,125 +32,127 @@ protected function _getElementHtml(\Magento\Framework\Data\Form\Element\Abstract
                     'value' => (bool)$partnerInfo['interestFreeMaxDuration']
-                );
+                ];
             }
 
             if (isset($partnerInfo['status'])) {
-                $list[] = array(
+                $list[] = [
                     'title' => __('Status'),
                     'value' => $partnerInfo['status']
-                );
+                ];
             }
 
             if (isset($partnerInfo['currency'])) {
-                $list[] = array(
+                $list[] = [
                     'title' => __('Currency'),
                     'value' => $partnerInfo['currency']
-                );
+                ];
             }
 
             if (isset($partnerInfo['minPaybackAmount'])) {
-                $list[] = array(
+                $list[] = [
                     'title' => __('Minimal payback amount'),
                     'value' => $partnerInfo['minPaybackAmount']
-                );
+                ];
             }
 
             if (isset($partnerInfo['paybackRate'])) {
-                $list[] = array(
+                $list[] = [
                     'title' => __('Payback rate'),
                     'value' => $partnerInfo['paybackRate']
-                );
+                ];
             }
 
             if (isset($partnerInfo['limit']['financing'])) {
-                $list[] = array(
+                $list[] = [
                     'title' => __('Financing limit'),
                     'value' => (int)$partnerInfo['limit']['financing']
-                );
+                ];
             }
 
             if (isset($partnerInfo['limit']['prepayment'])) {
-                $list[] = array(
+                $list[] = [
                     'title' => __('Prepayment limit'),
                     'value' => (int)$partnerInfo['limit']['prepayment']
-                );
+                ];
             }
 
             if (isset($partnerInfo['limit']['total'])) {
-                $list[] = array(
+                $list[] = [
                     'title' => __('Total limit'),
                     'value' => (int)$partnerInfo['limit']['total']
-                );
+                ];
             }
 
             if (isset($partnerInfo['interest']['nominal'])) {
-                $list[] = array(
+                $list[] = [
                     'title' => __('Interest nominal'),
                     'value' => $partnerInfo['interest']['nominal']['min']
                         . " - " . $partnerInfo['interest']['nominal']['max']
-                );
+                ];
             }
 
             if (isset($partnerInfo['interest'])) {
-                $list[] = array(
-                    'title' => __('Interest effective'),
-                    'value' => $partnerInfo['interest']['effective']['min']
-                        . " - " . $partnerInfo['interest']['effective']['max']
-                );
             }
+            $list[] = [
```

Comment:
All others are checked with isset first, is eff. interest left out on purpose?

Reply:
Thank you for the feed back. I missed that part, sorry. Fixed that.